dbadmin/testing_data.sql

```diff
@@ -0,0 +1,63 @@
+(10, 'sha256:47d1bcf86b085aacf96ad606aa44f6a9f67073a23643e7865a1e70761132cb8d978810363eb7da1580a59c5f0c0af767ba9431b84bad58f6843edc59e0561eba3685ce0f3feb95d022390d69d4eddccee902d276c1b3cf0c048fcae912485d83c6f85335c9f497862b0722f660966306b559f9ef9ce6d81d0c6c568fb09d04baef3d42e1677c55f5532d8c2b022ce1ac028519c65dc39badfa53fe035329f4ca61a3c6578a1b4a435d66568eef33d3a0af6edc2f3ee0fce69f9765c872fae0f007e67ce6875620bf6390e1c58b8ebc463eeb8272f403adeb4682652045e8409b8f94b2793e0842e73d390db93e70fee05109fde6ec134062755f055033a9cb3a9c941a06778fd6b6632e0be68c80bfd568c8a5d52fa1e4b78c45eb6a1db85d2283bc5748fc3079a43cc02c10a91122c31dd605e4e52303c710caa38578210199', 'e'),
+(11, 'sha256:410a55a2c85c7325e78b37f4a73e60610e6ffa18b6b1a1327a5b112ebaef3bd566dab2124b954d60c97dd84f21f53dd2457bb458d6ba3baa5814d00c875e908a6e1d8b60940f40282da254f147017590c01bf8fab28e63cef4cffaeb882d1c09aff761cdf4c506b13c7fe875fd06fa678f47c8f46d86c76fe295d67c19087f98696d9fe954f4afb1f1640dc85dbb8873f710f763de508973accc7578cd2ac0439aac38bed23a3ba5fd05c85774a25a497ae846a4721493971e63577617061c5f53a0f9220663aa2437993698dc875e02c00f3bc54d97a8783d240434b3c3750b17137a5aeaaeaff5be4f68a48d7bee002857955160e9742a6a4d0d5efb8d422db24cf38cbba6eaee01d13ba50086bc30768218ff0695fc09eac25dd6d5ada6b015ec73b3e1a50823f031a1e399c8d8db7b15d94ee748d7fba0a0ea34511b242a', 'a'),
+(12, 'sha256:8f252831f96d1b3a9c516e084302d723d98a57f9185f6b3d5e58a95efe0f04acd2070278402ef085c6a63455065e8c1c3b696cc696271db2393884c2c9fdd1c0c4ea5ebac1c390a04b7fed2a41bf1d38f512739f1d4f5c19edc6c0a4fae2e8970ebd8d87756f8c4939e941e82904966c7b007d73fd99b2abb4a7573cd968178992294bea4b50394ef78abd536480437e189976d8cf34b47c9dbb53f1de6c571d4739a3891af63c91958572f74b9e98a3648c6f72ac247eacdd99800a4c111bd04d55dd34ffdfad1de19ba69a480c61839d8ff87988b80c54e2ee562128bc5ad6b702512b5db7415fa232130628630d57493a3d076a6863f31db4376c04d3830bbdf5e93254da222459f5e840057764bfdcd46b34822aecf138eaf04a47eb9ceef66b12aa1e905e951d202e59910b9c022c006fdba557a35e847d5c6bc0a91d77', 'a');;
+
+INSERT INTO image_cve (image_id, cve_id) VALUES
+(1, 20),
+(2, 21),
+(3, 22),
+(3, 24),
+(4, 20),
+(4, 23),
+(5, 23),
+(6, 20),
+(7, 20),
+(8, 20),
+(9, 22),
+(10, 22),
+(11, 23),
+(11, 24);
+
+INSERT INTO cve (id, name, description, severity, cvss3_score, cvss3_metrics, cvss2_score, cvss2_metrics) VALUES
+(20, 'Sandstorm', 'Sandstorm mock CVE', 'High', NULL, NULL, 8, 'what'),
```

Comment:
The name in this context is the CVE identifier, I'd use names like `CVE-2022-0001` to be more explanatory.